data/functions/itemized.sql

```diff
@@ -15,3 +121,22 @@ begin
     perform retry_processing_schedule_b_records(:START_YEAR_AGGREGATE);
 end
 $$ language plpgsql;
+
+CREATE OR REPLACE FUNCTION rename_table_cascade(table_name TEXT) RETURNS VOID AS $$
```

Comment:
Ah, I did not realize that index names need to unique in the schema, not just in the table.